google-cloud-spanner/lib/google/cloud/spanner/database.rb

```diff
@@ -443,14 +463,48 @@ def database_operations filter: nil, page_size: nil
         #     backup = job.backup
         #   end
         #
-        def create_backup backup_id, expire_time, version_time: nil
+        # @example Create backup with encryption config
+        #   require "google/cloud/spanner"
+        #
+        #   spanner = Google::Cloud::Spanner.new
+        #   database = spanner.database "my-instance", "my-database"
+        #
+        #   kms_key_name = "projects/<project>/locations/<location>/keyRings/<key_ring>/cryptoKeys/<kms_key_name>"
+        #   encryption_config = {
+        #     kms_key_name: kms_key_name,
+        #     encryption_type: :CUSTOMER_MANAGED_ENCRYPTION
+        #   }
+        #   job = database.create_backup "my-backup",
+        #                                Time.now + 36000,
+        #                                encryption_config: encryption_config
+        #
+        #   job.done? #=> false
+        #   job.reload! # API call
+        #   job.done? #=> true
+        #
+        #   if job.error?
+        #     status = job.error
+        #   else
+        #     backup = job.backup
+        #   end
+        #
+        def create_backup backup_id, expire_time,
```

Comment:
What happens if we try to create a database / backup / restore with an invalid `encryption_type`. Something like:

```ruby
spanner = Google::Cloud::Spanner.new
database = spanner.database "my-instance", "my-database"
kms_key_name = "projects/<project>/locations/<location>/keyRings/<key_ring>/cryptoKeys/<kms_key_name>"
encryption_config = { encryption_type: :INVALID_ENCRYPTION }

job = database.create_backup "my-backup", Time.now + 36000, encryption_config: encryption_config
```

Reply:
`encryption_type` is enum  so if value is not present in enum then Protobuf will reject it with error.

`Unknown symbol value for enum field 'encryption_type'.`

Reply:
Cool, thanks for the information.